tools/ports/sdl2_gfx.py

```diff
@@ -16,8 +16,6 @@ def needed(settings):
 
 
 def get(ports, settings, shared):
-  sdl_build = os.path.join(ports.get_build_dir(), 'sdl2')
-  assert os.path.exists(sdl_build), 'You must use SDL2 to use SDL2_gfx'
```

Comment:
This doesn't work when the actual library build is deferred, but the dependency code in ports/__init__.py should ensure that the dependency is met.